dsc/src/subcommand.rs

```diff
@@ -606,7 +606,7 @@ fn list_resources(dsc: &mut DscManager, resource_name: &Option<String>, adapter_
 
             // if description is specified, skip if resource description does not contain it
             if description.is_some() &&
-                (manifest.description.is_none() | !manifest.description.unwrap_or_default().to_lowercase().contains(&description.as_ref().unwrap_or(&String::new()).to_lowercase())) {
+                (manifest.description.is_none() | !manifest.description.unwrap_or_default().to_lowercase().contains(&description.as_ref().unwrap_or(&&String::new()).to_lowercase())) {
```

Comment:
```suggestion
                (manifest.description.is_none() | !manifest.description.unwrap_or_default().to_lowercase().contains(description.unwrap_or(&String::new()).to_lowercase())) {
```
Is the double reference still necessary for `description` now that it's a `&String` when unwrapped?

Reply:
Good catch!  The double ref isn't needed, but `description` still needs to be a reference `.contains()` requires yet, but I was on auto-pilot making the changes and the `.as_ref()` which makes it a double ref isn't needed, so then it just needs to be `&String::new()`.  I'll make a fix.